lib/rules/call-indentation.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview Check indentation at the beginning and end of a function call
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+   var sourceCode = context.getSourceCode(),
+       lines = sourceCode.getLines();
+
+   //--------------------------------------------------------------------------
+   // Helpers
+   //--------------------------------------------------------------------------
+
+   function lineIndent(lineNumber) {
+      var line = lines[lineNumber - 1];
+
+      if (line === undefined) {
+         throw new Error('Invalid lineNumber ' + lineNumber + ' supplied to lineIndent');
+      }
+
+      return line[0] === ' ' ? line.match(/[^ ]/).index : 0;
```

Comment:
once again we're only checking for spaces ... and this is duplicate code. Please refactor to a common helper